pq-crypto/bike_r3/Makefile

```diff
@@ -17,7 +17,10 @@
 
 include ../../s2n.mk
 
-SRCS=$(wildcard *.c)
+# TODO: Add the x86_64 optimized implementations to the Makefile build system,
```

Comment:
We shouldn't add any new code without having proper build support for both CMake and Make. The github CI builds use the Make build system, so excluding all the new code from the Makefile means that the CI results don't actually tell us anything about whether the build is good or not. This is especially relevant to optimized code like this that is highly dependent on underlying architecture.

Reply:
Added new Makefile.